chapters/synchronous.tex

```diff
@@ -918,17 +934,14 @@ \subsection{Connected Components of the Equations and Variables Graph}\label{con
 Each use of \lstinline!time! is conceptually included as a separate variable in this analysis, $\mathit{time}_i$ with $\text{\lstinline!der!}(\mathit{time}_{i}) = 1$.
 \begin{nonnormative}
 This means that \lstinline!time! can be used in different partitions without any restrictions.
-Additionally, it means that every partition directly referencing \lstinline!time! contains a call to \lstinline!der!.
+Additionally, it means that every sub-partition directly referencing \lstinline!time! contains a call to \lstinline!der!.
 \end{nonnormative}
 
-\subsection{Base-Clock Partitioning}\label{base-clock-partitioning}
+\subsection{Base-Partitioning}\label{base-clock-partitioning}
 
-The goal is to identify all clocked equations and variables that should
-be executed together in the same task, as well as to identify the
-continuous-time partition.
+The goal is to identify all clocked equations and variables that should be executed together in the same task, as well as to identify the continuous-time base-partition.
 
-The base-clock partitioning is performed with base-clock inference which
-uses the following incidence definition:
+The base-partitioning is performed with base-clock inference which uses the following incidence definition:
```

Comment:
Don't break the paragraph after this line, just break the line (can't make Github suggestion for that).

Reply:
Done